### PR TITLE
Travis integration for unit tests

### PR DESCRIPTION
**Pull request by ravisantoshgudimetla**
/cc @aveshagarwal. This is for travis ci integration. I can include a code coverage badge as well from coversall.io. Let me know what you think about it?

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+go:
+  - 1.8.3
+
+before_install:
+  - sudo apt-get install libseccomp-dev -qq
+
+before_script:
+  - mkdir -p $HOME/gopath/src/github.com/aveshagarwal
+  - mv $TRAVIS_BUILD_DIR $HOME/gopath/src/github.com/aveshagarwal
```
Comment (aveshagarwal) [inline .travis.yml]: could you change these paths to kubernetes-incubator now?

Comment (ravisantoshgudimetla) [reply .travis.yml]: I can do that. In fact, I was pinging you today morning for the same reason. I think, you are working on a PR where you want to change name from rescheduler to descheduler, we can include your name removal as part of that. But for this PR, I will change and re-run the tests and see.

**Comment by ravisantoshgudimetla** [comment]
So, after changing to kube-incubator, the travis-ci is failing. 

https://travis-ci.org/ravisantoshgudimetla/rescheduler/builds/275203057?utm_source=github_status&utm_medium=notification

**Comment by aveshagarwal** [comment]
@ravisantoshgudimetla could you check if its failing or not?

**Comment by ravisantoshgudimetla** [comment]
@aveshagarwal Done. You may need to give Travis CI, access to repository. This is how I did for my fork of this repo. 

https://github.com/mbonaci/mbo-storm/wiki/Integrate-Travis-CI-with-your-GitHub-repo

**Pull request merged**